cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -191,37 +197,89 @@ def enable_disable_migration_ui(appliance):
 @pytest.mark.parametrize('form_data_single_datastore', [['nfs', 'nfs'],
                             ['nfs', 'iscsi'], ['iscsi', 'iscsi']], indirect=True)
 def test_single_datastore_single_vm_mapping_crud(appliance, enable_disable_migration_ui,
-                                            create_provider, form_data_single_datastore):
-    # TODO: Add "Delete" method call.This test case does not support update/delete
-    # as update is not a supported feature for mapping,
-    # and delete is not supported in our automation framework.
+                                            providers, form_data_single_datastore,
+                                            soft_assert):
+    # TODO: This test case does not support update
+    # as update is not a supported feature for mapping.
     infrastructure_mapping_collection = appliance.collections.v2v_mappings
     mapping = infrastructure_mapping_collection.create(form_data_single_datastore)
     view = navigate_to(infrastructure_mapping_collection, 'All', wait_for_view=True)
     assert mapping.name in view.infra_mapping_list.read()
+    mapping_list = view.infra_mapping_list
+    mapping_list.delete_mapping(mapping.name)
+    view.browser.refresh()
+    view.wait_displayed()
+    try:
```

Comment:
**REQUIRED:** I don't think this is enough. I think that we should minimally, assert that there is some kind of table there, or validate the table is not there in the exception handling. Rather than just catching the NSEE and asserting True.

Reply:
@psav, it throws NSEE when the table (or rather list) that contains the Infra Mappings is empty, as the element containing the list is hidden. So it does make sense to assert True if element not found for "infra-mappings-list" as empty list means deletion succeeded.